nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -122,7 +125,8 @@ def create_clinical_review_task(self, activity):
         """
         clinical_review_model = \
             self.pool['nh.clinical.notification.clinical_review']
-        due_date = datetime.now().strftime(DTF)
+        datetime_utils_model = self.pool['datetime_utils']
+        due_date = datetime_utils_model.get_current_time().strftime(DTF)
```

Comment:
Can achieve the same as this by doing `datetime_utils_model.get_current_time(as_string=True)`.

Reply:
Yep